.flowconfig

```diff
@@ -5,46 +5,69 @@
 ; Ignore "BUCK" generated dirs
 <PROJECT_ROOT>/\.buckd/
 
-; Ignore unexpected extra "@providesModule"
-.*/node_modules/.*/node_modules/fbjs/.*
-
-; Ignore duplicate module providers
-; For RN Apps installed via npm, "Libraries" folder is inside
-; "node_modules/react-native" but in the source repo it is in the root
-.*/Libraries/react-native/React.js
-
 ; Ignore polyfills
-.*/Libraries/polyfills/.*
+node_modules/react-native/Libraries/polyfills/.*
+
+; Flow doesn't support platforms
+.*/Libraries/Utilities/LoadingView.js
+[untyped]
+.*/node_modules/@react-native-community/cli/.*/.*
 
 [include]
 
 [libs]
-node_modules/react-native/Libraries/react-native/react-native-interface.js
+node_modules/react-native/interface.js
 node_modules/react-native/flow/
 
 [options]
 emoji=true
 esproposal.optional_chaining=enable
 esproposal.nullish_coalescing=enable
 
-module.system=haste
+exact_by_default=true
+
+module.file_ext=.js
+module.file_ext=.json
+module.file_ext=.ios.js
+
+module.system.haste.paths.whitelist=<PROJECT_ROOT>/node_modules/react-native/Libraries/.*
+module.system.haste.paths.whitelist=<PROJECT_ROOT>/node_modules/react-native/RNTester/.*
+module.system.haste.paths.whitelist=<PROJECT_ROOT>/node_modules/react-native/IntegrationTests/.*
+module.system.haste.paths.blacklist=<PROJECT_ROOT>/node_modules/react-native/Libraries/react-native/react-native-implementation.js
+module.system.haste.paths.blacklist=<PROJECT_ROOT>/node_modules/react-native/Libraries/Animated/src/polyfills/.*
 
 munge_underscores=true
 
-module.name_mapper='^[./a-zA-Z0-9$_-]+\.\(bmp\|gif\|jpg\|jpeg\|png\|psd\|svg\|webp\|m4v\|mov\|mp4\|mpeg\|mpg\|webm\|aac\|aiff\|caf\|m4a\|mp3\|wav\|html\|pdf\)$' -> 'RelativeImageStub'
+module.name_mapper='^react-native/\(.*\)$' -> '<PROJECT_ROOT>/node_modules/react-native/\1'
+module.name_mapper='^@?[./a-zA-Z0-9$_-]+\.\(bmp\|gif\|jpg\|jpeg\|png\|psd\|svg\|webp\|m4v\|mov\|mp4\|mpeg\|mpg\|webm\|aac\|aiff\|caf\|m4a\|mp3\|wav\|html\|pdf\)$' -> '<PROJECT_ROOT>/node_modules/react-native/Libraries/Image/RelativeImageStub'
```

Comment:
How does `.flowconfig` work? Isn't L42 overwriting L41? Same question for L33-35.

Reply:
this is how it is shown in the react-native migration tool (an official tool). 

See [the entire file](https://raw.githubusercontent.com/react-native-community/rn-diff-purge/release/0.64.2/RnDiffApp/.flowconfig) - you can get the file by [opening the tool](https://react-native-community.github.io/upgrade-helper/?from=0.63.0&to=0.64.2) and using the "View file" for the `.flowconfig`


As of the lines 33-35... I can't find them in the tool. I can't remember where I got them from 🤔  I will take a look at that

Reply:
According to the docs, `module.name_mapper` can be set multiple times - [see here](a85ca06dd674b81c6ca5c0478882bbb744c79b8c). I think this, plus the fact that this is the way it should be according to react-native, is enough to leave it like that 😅 

Regarding lines 33-35, apparently got added when migrating from [0.59 to 0.60](https://react-native-community.github.io/upgrade-helper/?from=0.59.0&to=0.60.0), but then removed when migrating from [0.60 to 0.61](https://react-native-community.github.io/upgrade-helper/?from=0.60.0&to=0.61.0) and I failed to do so. So I removed them in https://github.com/autonomoussoftware/metronome-wallet-mobile/pull/128/commits/b03cd0e30adf03f9e208825138bd267b11c15f4d